docs/source/markdown/podman-rm.1.md

```diff
@@ -22,6 +22,10 @@ Remove all containers.  Can be used in conjunction with -f as well.
 
 Read container ID from the specified file and remove the container.  Can be specified multiple times.
 
+**--ignore-no-such-container**
+
+Ignore errors when specified containers are not in the container store.
```

Comment:
I think a little more background would be useful.  Borrowing from your commit message, perhaps something like:
```
Ignore errors when specified containers are no longer in the container store.  This type of error can occur after a user removes a container without using a podman command.
```
Similar change in stop.